spec/eofv0_verkle.md

```diff
@@ -105,6 +105,154 @@ The same as above except encode the values as 6-bit numbers
 (minimum number of bits needed for encoding `32`).
 Such encoding lowers the size overhead from 3.1% to 2.3%.
 
+### Encode only invalid pushdata (dense encoding)
+
+Alternate option is instead of encoding all valid `JUMPDEST` locations, to only encode invalid ones.
+
+This is beneficial if our assumption is correct that most contracts only contain a limited number
+of offending cases. Our initial analysis suggests this is the case, e.g. Uniswap router has 9 cases,
+one of the Arbitrum validator contracts has 6 cases.
+
+Since Solidity contracts have a trailing metadata, which contains a Keccak-256 (32-byte) hash of the
+source, there is a 12% probability ($1 - (255/256)^{32}$) that at least one of the bytes of the hash
+will contain the `0x5b` value, which gives our minimum probability of having at least one invalid
+`JUMPDEST` in the contract.
+
+Let's create a map of `invalid_jumpdests[chunk_no] = position_in_chunk`. We can densely encode this
```

Comment:
I think the `position_in_chunk` is not correct because it doesn't work for multiple invalid JUMPDESTs in a chunk.
As we discussed: we want to encode a piece of information allowing to perform local jumpdest analysis of the offending chunk: _first instructions offset_ of _first valid jumpdest_.

Reply:
Yes, it's `fio`. Can rename it.